stdlib/3.4/asyncio/events.pyi

```diff
@@ -71,39 +80,52 @@ class AbstractEventLoop(metaclass=ABCMeta):
     @abstractmethod
     def call_soon_threadsafe(self, callback: Callable[..., Any], *args: Any) -> Handle: ...
     @abstractmethod
+    @coroutine
     def run_in_executor(self, executor: Any,
         callback: Callable[..., Any], *args: Any) -> Future[Any]: ...
     @abstractmethod
     def set_default_executor(self, executor: Any) -> None: ...
     # Network I/O methods returning Futures.
     @abstractmethod
+    @coroutine
     def getaddrinfo(self, host: str, port: int, *,
         family: int = ..., type: int = ..., proto: int = ..., flags: int = ...) -> Future[List[Tuple[int, int, int, str, tuple]]]: ...
     @abstractmethod
+    @coroutine
     def getnameinfo(self, sockaddr: tuple, flags: int = ...) -> Future[Tuple[str, int]]: ...
     @abstractmethod
+    @coroutine
     def create_connection(self, protocol_factory: Any, host: str = ..., port: int = ..., *,
                           ssl: Any = ..., family: int = ..., proto: int = ..., flags: int = ..., sock: Any = ...,
                           local_addr: str = ..., server_hostname: str = ...) -> tuple: ...
```

Comment:
I wanted to keep this out of this PR to avoid doing too much at once. There's a lot of Anys in this file that aren't necessary; I can fix those in this PR too if you prefer.